fiftyone/brain/__init__.py

```diff
@@ -712,103 +712,98 @@ def compute_exact_duplicates(
+            detecting leaks. Values in ``[0.1, 0.25]`` work well for the
+            default setup
+        embeddings (None): if no ``model`` is provided, this argument specifies
+            pre-computed embeddings to use, which can be any of the following:
+
+            -   a ``num_samples x num_dims`` array of embeddings
+            -   if ``roi_field`` is specified,  a dict mapping sample IDs to
+                ``num_patches x num_dims`` arrays of patch embeddings
+            -   the name of a dataset field containing the embeddings to use
+
+            If a ``model`` is provided, this argument specifies the name of a
+            field in which to store the computed embeddings
+        similarity_index (None): a
+            :class:`fiftyone.brain.similarity.SimilarityIndex` or the brain key
+            of a similarity index to use to load pre-computed embeddings
         model (None): a :class:`fiftyone.core.models.Model` or the name of a
             model from the
-            `FiftyOne Model Zoo <https://docs.voxel51.com/user_guide/model_zoo/index.html>`_
-            to use, or that was already used, to generate embeddings. The model
-            must expose embeddings (``model.has_embeddings = True``)
+            `FiftyOne Model Zoo <https://docs.voxel51.com/user_guide/model_zoo/models.html>`_
+            to use to generate embeddings. The model must expose embeddings
+            (``model.has_embeddings = True``)
         model_kwargs (None): a dictionary of optional keyword arguments to pass
             to the model's ``Config`` when a model name is provided
-        similarity_backend: string, the similarity backend to use. The supported values are
-            ``fiftyone.brain.brain_config.similarity_backends.keys()`` and the
-            default is
-            ``fiftyone.brain.brain_config.default_similarity_backend``
-        similarity_config_dict: dict, used to build the similarity backend. Arguments take
-            precedence over the values in the dict (e.g. model)
+        batch_size (None): a batch size to use when computing embeddings. Only
+            applicable when a ``model`` is provided
+        num_workers (None): the number of workers to use when loading images.
+            Only applicable when a Torch-based model is being used to compute
+            embeddings
+        skip_failures (True): whether to gracefully continue without raising an
+            error if embeddings cannot be generated for a sample
+        progress (None): whether to render a progress bar (True/False), use the
+            default value ``fiftyone.config.show_progress_bars`` (None), or a
+            progress callback function to invoke instead
 
     Returns:
-        a :class:`fiftyone.brain.internal.core.leaky_splits.LeakySplitsIndex`,
-        a :class:`fiftyone.core.view.DatasetView`
+        a :class:`fiftyone.brain.internal.core.leaky_splits.LeakySplitsIndex`
```

Comment:
This is a breaking change to the interface: users must now run:

```py
index = fob.compute_leaky_splits(dataset, ...)
leaks = index.leaks_view()
```

rather than

```py
index, leaks = fob.compute_leaky_splits(dataset, ...)
```

Reasoning here is for consistency with other brain methods, and also because there's no guarantee I want `leaks_view`: maybe I just want to access `index.leak_ids` or `index.no_leaks_view()`, for example.

I updated the Brain docs in https://github.com/voxel51/fiftyone/pull/5188 to account for this change, but I assume the blog post will need tweaked as well.